glean-core/src/coverage.rs

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
```

Comment:
Can we add some module docs here (`//!`) of what this does. Calling out the environment variable and the format in which the file is written ("one metric id per line")